pkg/api/db.go

```diff
@@ -73,6 +73,17 @@ func CreateVolumeDBEntry(ctx *c.Context, in *model.VolumeSpec) (*model.VolumeSpe
 	return db.C.CreateVolume(ctx, in)
 }
 
+func CreateVolumeError(ctx *c.Context, in *model.VolumeSpec) {
```

Comment:
Can we make this function generic for any Volume Error ops? AFAIU, it's not specific to create volume. VolumeErrorHandle() ?